algorithms/kernel/svm/svm_train_thunder_impl.i

```diff
@@ -101,25 +100,60 @@ services::Status SVMTrainImpl<thunder, algorithmFPType, ParameterType, cpu>::com
     DAAL_CHECK_MALLOC(yTArray.get());
     algorithmFPType * const y = yTArray.get();
 
+    SafeStatus safeStat;
+
+    size_t nNonZeroWeights = nVectors;
     {
-        ReadColumns<algorithmFPType, cpu> mtY(yTable, 0, 0, nVectors);
-        DAAL_CHECK_BLOCK_STATUS(mtY);
-        const algorithmFPType * const yIn = mtY.get();
-
-        ReadColumns<algorithmFPType, cpu> mtW(wTable.get(), 0, 0, nVectors);
-        DAAL_CHECK_BLOCK_STATUS(mtW);
-        const algorithmFPType * weights = mtW.get();
-        // gradi = -yi; ai = 0; ci = c*wi
-        for (size_t i = 0; i < nVectors; i++)
+        const size_t blockSize = 16384;
```

Comment:
Does it calculated by experiment?

Reply:
Copying is a fairly quick operation within a single core. I would like to reduce the number of blocks - by increasing the size of the block. Moreover, this code is called in parallel with a multiclass. This size has been tested experimentally.